src/overlays/actors/ovl_En_Tru/z_en_tru.c

```diff
@@ -307,8 +326,8 @@ void func_80A85F84(EnTruUnkStruct* arg0, PlayState* play) {
 
             gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(play->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
             idx = ((f32)arg0->unk_02 / arg0->unk_01) * 8.0f;
-            gSPSegment(POLY_XLU_DISP++, 0x08, Lib_SegmentedToVirtual(sDustTextures[idx]));
-            gSPDisplayList(POLY_XLU_DISP++, object_tru_DL_01A830);
+            gSPSegment(POLY_XLU_DISP++, 0x08, Lib_SegmentedToVirtual(sDustTextures[(idx)]));
```

Comment:
why the extra `()`?

Reply:
Whoops. I was messing with the function, and must have forgot to remove the () again.